packages/utils/src/useColumnManager/helper.js

```diff
@@ -0,0 +1,8 @@
+import get from 'lodash/get';
+
+export const filterColumnsBySelected = (columns, selected, selectProp) => columns.filter((column) => selected.includes(get(column, selectProp)));
+
+const managableColumns = (columns) =>
```

Comment:
I would rather pass in here a default value
```suggestion
const managableColumns = (columns = []) =>
```